setup.py

```diff
@@ -146,28 +146,28 @@ def build_libraries(self, libraries):
                                            
 
 setup(
-    name = 'RAT',
```

Comment:
Is this intended or did your IDE remove the spaces somehow? We need to get a linter/formatter soon to avoid unnecessary reformatting as the code grows

Reply:
Yes, as I was updating this file pydantic moaned at me about these spaces. Agreed on sorting a linter soon.

Reply:
Does pydantic have a linter? how is it concerned with the setup.py

Reply:
D'oh - pyCHARM is what I meant.